cfme/tests/configure/test_access_control.py

```diff
@@ -242,6 +247,26 @@ def tenant_custom_role(appliance, request, provider):
     tenant_role.delete_if_exists()
 
 
+@pytest.fixture()
+def user_catalog(appliance, request, ansible_catalog_item):
+    group = appliance.rest_api.collections.groups.get(description="EvmGroup-user_self_service")
```

Comment:
Optional - You can just assign group description to a variable instead of instantiating the group since you're only using group description.